server/src/services/job.service.spec.ts

```diff
@@ -286,23 +286,23 @@ describe(JobService.name, () => {
         jobs: [],
       },
       {
-        item: { name: JobName.GENERATE_JPEG_THUMBNAIL, data: { id: 'asset-1' } },
-        jobs: [JobName.GENERATE_WEBP_THUMBNAIL, JobName.GENERATE_THUMBHASH_THUMBNAIL],
+        item: { name: JobName.GENERATE_THUMBNAIL, data: { id: 'asset-1' } },
+        jobs: [JobName.GENERATE_PREVIEW, JobName.GENERATE_THUMBHASH_THUMBNAIL],
```

Comment:
I believe that's mixed up. `GENERATE_JPEG_THUMBNAIL` should correspond to `GENERATE_PREVIEW` and `GENERATE_WEBP_THUMBNAIL` should correspond to `GENERATE_THUMBNAIL`, right?